dotCMS/src/integration-test/java/com/dotcms/datagen/FileAssetDataGen.java

```diff
@@ -32,4 +36,24 @@ private FileAssetDataGen(final File file) throws DotDataException, DotSecurityEx
         setProperty(FileAssetAPI.BINARY_FIELD, file);
     }
 
+    public static Contentlet createFileAsset(
+            final Folder folder,
+            final String fileName,
+            final String suffix)
+            throws IOException, DotSecurityException, DotDataException {
+
+        return  createFileAsset(folder, fileName, suffix, "helloworld");
+    }
+
+    private static Contentlet createFileAsset(
+            final Folder folder,
+            final String fileName,
+            final String suffix,
+            final String content)
+            throws IOException, DotSecurityException, DotDataException {
+        final java.io.File file = java.io.File.createTempFile(fileName, suffix);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'java.io.File.createTempFile' due to existing import 'java.io.File'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5280157)